src/components/PeopleTable.tsx

```diff
@@ -10,635 +108,136 @@ export const PeopleTable = () => {
           <th>
             <span className="is-flex is-flex-wrap-nowrap">
               Name
-              <a href="#/people?sort=name">
+              <SearchLink
+                params={{
+                  sort: 'name',
+                  order: sortOrder === 'asc' ? 'desc' : 'asc',
+                }}
+                onClick={() => handleSort('name')}
+              >
                 <span className="icon">
-                  <i className="fas fa-sort" />
+                  <i
+                    className={`fas fa-sort-${sortOrder === 'asc' ? 'up' : 'down'}`}
```

Comment:
The class name for the sort icon is determined by the `sortOrder`. Ensure that the icon correctly reflects the current sort order by verifying that `sortOrder` is properly initialized and updated. Consider using a default icon state when `sortOrder` is null.